source/User_Guide/Marketing_Campaigns/senders.md

```diff
@@ -22,16 +22,15 @@ Before you begin, go to your SendGrid [account settings]({{site.app_url}}/user/a
 Adding a Sender
 {% endanchor h2 %}
 
-1. Click **Create New Sender**
```

Comment:
I feel like this sort of drops into the middle of the task, and is missing the "navigate to this page, click on this..." context